draft-ietf-ppm-dap.md

```diff
@@ -624,87 +622,77 @@ Helper:
+also be one of the Aggregators.
 
 In the course of a measurement task ({{task-configuration}}), each Client
-records its own measurement value(s), packages them up into a report, and sends
-them to the Leader. Each share is encrypted to only one of the two Aggregators
-so that even though they pass through the Leader, the Leader is unable to see or
-modify the Helper's share. Depending on the task, the Client may only send one
-report or may send many reports over time.
+records its own measurement, packages it up into a report, and sends it to the
+the Leader. Each share is encrypted to only one of the two Aggregators so that
+even though they pass through the Leader, the Leader is unable to see or modify
+the Helper's share. Depending on the task, the Client may only send one report
+or may send many reports over time.
 
 The Leader distributes the shares to the Helper and orchestrates the process of
-verifying them (see {{validating-inputs}}) and assembling them into an aggregate
-result for the Collector. Depending on the VDAF, it may be possible to
-incrementally process each report as it is uploaded, or it may be necessary to
-wait until the Collector transmits a collection request before processing can
-begin.
+verifying them and assembling them into a aggregate shares for the Collector.
+Depending on the VDAF, it may be possible to process each report as it is
+uploaded, or it may be necessary to wait until the Collector transmits a
+collection request before processing can begin.
 
 ## Validating Measurements {#validating-inputs}
 
-An essential task of any data collection pipeline is ensuring that the data
+An essential goal of any data collection pipeline is ensuring that the data
 being aggregated is "valid". For example, each measurement might be expected to
-be a number between `0` and `10`. In DAP, input validation is complicated by the
+be a number between 0 and 10. In DAP, input validation is complicated by the
 fact that none of the entities other than the Client ever sees that Client's
 plaintext measurement. To an Aggregator, a secret share of a valid measurement
 is indistinguishable from a secret share of an invalid measurement.
 
-In DAP, input validation is accomplished by an interactive computation between
-the Leader and Helper. At the beginning of this computation, each Aggregator is
-in possession of an input share uploaded by the Client. At the end of the
-computation, each Aggregator is in possession of either an "output share" that
-is ready to be aggregated or an indication that the underlying data is invalid,
-in which case the report is rejected.
+DAP validates inputs using an interactive computation between the Leader and
+Helper. At the beginning of this computation, each Aggregator holds an input
+share uploaded by the Client. At the end of the computation, each Aggregator
+either holds an output share that is ready to be aggregated or rejects the
```

Comment:
Having an active and passive verb on either side of this "either ... or" feels unusual to me
```suggestion
either obtains an output share that is ready to be aggregated or rejects the
```